src/BCEngineMCMC.cxx

```diff
@@ -2392,106 +2374,108 @@ void BCEngineMCMC::MCMCInitialize()
     // initialize markov chain positions
     switch (fInitialPositionScheme) {
 
-        // keep previous values
-        case kInitContinue : {
-            throw std::runtime_error("BCEngineMCMC::MCMCInitialize : Continuing chains not yet supported. Sorry!");
-
-            // check position vector size
-            if (fMCMCx.size() != fMCMCNChains)
-                throw std::runtime_error("BCEngineMCMC::MCMCInitialize : Number of chains has been changed; cannot continue previous chains.");
-
-            // else do nothing --- continue chains
-            break;
+    // keep previous values
+    case kInitContinue : {
+        throw std::runtime_error("BCEngineMCMC::MCMCInitialize : Continuing chains not yet supported. Sorry!");
```

Comment:
Didn't @tpoeschl have a patch for continuing?

Reply:
He's handed me his code. I will see how easy it is to integrate. I think it mostly involves adding some getters and setters.